R/data.R

```diff
@@ -52,7 +53,7 @@
 "ecpe_qmatrix"
 
 
-#' MacReady & Dayton (1977) Multiplication Data
+#' MacReady & Dayton (1977) Multiplication Data  `r # nolint: sentence_case.`
```

Comment:
This lint is ignored using the `` `r # nolint` `` tag. This will disable lints for the entire Rd element. I'm not super crazy about the syntax or how broadly it applies, but it is entirely transparent in the generated Rd file, so it should be more portable than `\nolint`.